common/src/java/org/apache/hive/http/HttpServer.java

```diff
@@ -652,6 +652,8 @@ ServerConnector createAndAddChannelConnector(int queueSize, Builder b) {
 
     final HttpConfiguration conf = new HttpConfiguration();
     conf.setRequestHeaderSize(1024*64);
+    conf.setSendServerVersion(false);
+    conf.setSendXPoweredBy(false);
```

Comment:
I'm checking HS2. The current response header is like this.

```
HTTP/1.1 200 OK
Date: Tue, 06 May 2025 07:12:49 GMT
Content-Type: text/html;charset=utf-8
X-Content-Type-Options: nosniff
X-FRAME-OPTIONS: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Content-Length: 3815
Server: Jetty(9.4.57.v20241219)
```

With HIVE-28653,

```
HTTP/1.1 200 OK
Date: Tue, 06 May 2025 07:32:43 GMT
Content-Type: text/html;charset=utf-8
X-Content-Type-Options: nosniff
X-FRAME-OPTIONS: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Content-Length: 3815
```

It looks like X-Powerered-By is originally false.